yarn-project/noir-protocol-circuits/src/crates/types/src/constants.nr

```diff
@@ -110,13 +114,13 @@ global CALL_CONTEXT_LENGTH: Field = 8;
 global GLOBAL_VARIABLES_LENGTH: Field = 4;
 global PARTIAL_STATE_REFERENCE_LENGTH: Field = 8;
 global STATE_REFERENCE_LENGTH: Field = 10; // 2 for snap + 8 for partial
-global HEADER_LENGTH: Field = 18; // 2 for last_archive, 2 for body hash, 10 for state reference, 4 for global vars 
+global HEADER_LENGTH: Field = 18; // 2 for last_archive, 2 for body hash, 10 for state reference, 4 for global vars
 global FUNCTION_DATA_LENGTH: Field = 4;
 global CONTRACT_DEPLOYMENT_DATA_LENGTH: Field = 6;
 // Change this ONLY if you have changed the PrivateCircuitPublicInputs structure.
 // In other words, if the structure/size of the public inputs of a function call changes then we should change this
-// constant as well PRIVATE_CALL_STACK_ITEM_LENGTH
-global PRIVATE_CIRCUIT_PUBLIC_INPUTS_LENGTH: Field = 204;
+// constant as well CALL_PRIVATE_FUNCTION_RETURN_SIZE
+global PRIVATE_CIRCUIT_PUBLIC_INPUTS_LENGTH: Field = 206;
```

Comment:
Why 2 more? isn't just one new field?

Reply:
Thank you Alvaro! This was a mistake while resolving merge conflicts.

Reply:
That's what I'm trying to figure out :laughing:. This was after a merge conflict